src/board/board.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk";
+import {drawChalkText, tileDataInterface} from "../tile/tile";
+
+type boardData = Array<Array<undefined|tileDataInterface>>
+
+const BLANK_CHAR = chalk.black(' ')
+
+/**
+ *  @description Main entrypoint of kamon.js script.
+ *  It will store states of current game, display it, check victory conditions, etc.
+ */
+
+/**
+ * @description This will draw a line of tiles in console from an array of tiles.
+ */
```

Comment:
useless comment. If the function name is clear enough, you don't need it.

Reply:
Done